Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/Faq.rst

```diff
@@ -78,6 +78,33 @@ You need to exlude those in your `.crowdin.yaml` configuration which can be foun
 
 More information can be found in the documtation on crowdin: https://support.crowdin.com/configuration-file/
 
+How can I migrate Translations from Pootle?
+-------------------------------------------
+If translations exist on Pootle there is no need to retranslate everything on Crowdin again - you can import those.
+
+
+#. **Fetch translations**
+   Download the translations you need. The given example will download the german translations of the extension *powermail*:
```

Comment:
languages like "German" have to be written with a capital letter. See https://english.stackexchange.com/questions/381/when-should-the-word-english-be-capitalized